Index: models/search/CustomerSearch.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- models/search/CustomerSearch.php	(revision 828b3c8017cbe9e106d74177b7f0514427b727b8)
+++ models/search/CustomerSearch.php	(date 1562674519000)
@@ -95,7 +95,7 @@
         }
         
         finalAction:
-        $query->orderBy([new \yii\db\Expression("FIELD(status_id, '1', '3', '2')")]);
+        $query->orderBy([new \yii\db\Expression("FIELD(status_id, '1', '3', '2')")])->addOrderBy(['updated_at' => SORT_ASC]);
         $dataProvider = new ActiveDataProvider([
             'query' => $query,
             'pagination' => [
